### ISSUE TITLE
Alert: Close button accessibility issues

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2082]
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-web-components/blob/main/CONTRIBUTING.md)

### Describe the issue

Tabbing to the close button of an alert is simply announced as 'clickable'. There is no text, aria-label and there is no role defined.

To fix the issue the `icon-close-container` div should have aria-label="Dismiss alert" and `role="button"`.

### Reduced test cases

I tested this URL: https://modus-web-components.trimble.com/?path=/docs/components-alert--default with NVDA on Windows 11.

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What is the issue regarding ?

@trimble-oss/modus-web-components

### What version of npm package are you using ?

v.0.13.0

### Priority

Medium

### What product/project are you using Modus Components for ?

Modus

### What is your team/division name ?

Modus

### Are you willing to contribute ?

Yes

### Are you using Modus Web Components in production ?

https://modus-web-components.trimble.com/
